src/actions/guides/database/querying.cr

```diff
@@ -7,293 +7,79 @@ class Guides::Database::Querying < GuideAction
-    ```
-
-    ## Type specific query methods
-
-    The previous section mostly focused on simple equality queries, but you can also
-    do more advanced queries depending on the column type.
-
-    ### All types
-
-    * `is`
-    * `desc_order`
-    * `asc_order`
-
-    ### String
-
-    * `like`
-    * `ilike`
-    * `lower`
-
-    ### Int32 and Time
-
-    * `gt`
-    * `gte`
-    * `lt`
-    * `lte`
-
-    ### Examples
-
-    ```crystal
-    UserQuery.new.age.gte(18).name.lower.is("sally")
-    ```
-
-    ## Using scopes
-
-    Generally it's best to name your query methods so they are easy to understand
-    and reuse across your app. In Lucky, this is done by extracting methods on the
-    query object.
-
-    Here's an example:
-
-    ```crystal
-    class UserQuery < User::BaseQuery
-      def adults
-        age.gte(18)
-      end
-
-      def search(name)
-        ilike("\#{name}%")
-      end
-    end
-
-    UserQuery.new.adults.search("Sal")
-    ```
-
-    ## Preloading associations
-
-    In development and test environemnts Lucky requries preloading associations.
-    If you forget to preload an association, a runtime error will be raised when
-    you try to access it. In production, the association will be lazy loaded so
-    that users do not see errors.
-
-    This solution means you will find N+1 queries in development and test, and
-    users will never see an error in production.
+    Some where query examples.
 
-    To preload, just call `preload_{association name}` on the query:
+    ### A = B
 
-    ```crystal
-    PostQuery.new.preload_comments
-    ```
+    ### A != B
 
-    ### Customizing how associations are preloaded
+    ### A gt/lt B
 
-    Sometimes you want to order preloads, or add where clauses. To do this, use
-    the `preload_{{ association_name }}` method on the query and pass a query
-    object for the association.
+    ### A in / not in (B)
 
-    ```crystal
-    PostQuery.new.preload_comments(CommentQuery.new.published(true))
-    ```
+    ### A like / iLike B
 
-    This is also how you would do nested preloads:
+    ## Ordering
 
-    ```crystal
-    # Preload the post's comments, and the comment's author
-    PostQuery.new.preload_comments(CommentQuery.new.preload_author)
-    ```
+    ASC / DESC methods
 
-    > Note that you can only pass query objects to `preload` if the association
-    is defined, otherwise you will get a type error.
+    ## Pagination
 
-    ## Loading associations without preloading
+    ### Limit
 
-    Sometimes you have a single model and don't need to preload items. Or maybe
-    you *can't* preload because the model record is already loaded. In those cases
-    you can use the association name with `!`:
+    ### Offset
 
-    ```crystal
-    post = PostQuery.first
-    # Returns the associated author and does not trigger a preload error
-    post.author!
-    ```
+    ## Associations and Joins
 
-    ## Joining and querying associations
+    ### Preloading
```

Comment:
Preloading should probably be `##` because it is a pretty big/important concept that could use a lot more examples and explanations